harvester_e2e_tests/integrations/test_4_vm_backup_restore.py

```diff
@@ -562,6 +562,70 @@ def test_restore_with_invalid_name(self, api_client, backup_config, base_vm_with
         code, data = api_client.backups.restore(unique_vm_name, spec)
         assert 422 == code, (code, data)
 
+    @pytest.mark.skip_version_before('v1.1.2', 'v1.2.1')
```

Comment:
The source issue harvester/harvester/issues/4307 resolved on `v1.3.0` and backport to `v1.2.2`. Should it be `skip_version_before("v1.2.2")`?

And as discussion before (https://github.com/harvester/tests/pull/1118#pullrequestreview-1897714128), we should use `@pytest.mark.skip_version_if` to avoid ambiguous `before`/`after`.